canonical-web-and-design/ubuntu.com
# 19.10 release

## Done

- Changes to the site to support the Ubuntu 19.10 (Eoan Ermine) release

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Look at the homepage, 19.10 webinar and all the download pages
